pam/integration-tests/cli-integration_test.go

```diff
@@ -0,0 +1,255 @@
+package main_test
+
+import (
```

Comment:
Since we are already inside the `integration-tests/` dir, maybe just use `cli_test.go` filename?